robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Allow: /
```

Comment:
There is no `Allow` field in the robots specs as per http://www.robotstxt.org/robotstxt.html ... 

Let's see whether we have some set of pages to restrict; that would be the cleanest solution to the no-allow problem.  

Maybe `/api/...` URLs, provided that we expose things via `/record/NNN/export/...` and `/record/NNN/files/FFF` kind of URLs?

Reply:
Depending on the #2247, yes, I'd say let's disallow `/api` for now.